workflow-job

```diff
@@ -702,20 +708,45 @@ def deployGuiceCassandra() {
     deployingGuiceCassandra.success()
 }
 
+def readBlobStoreImplementation() {
+    blobStoreFileName = 'dockerfiles/run/guice/cassandra-rabbitmq-ldap/destination/conf/blob.properties'
+    if (!fileExists(blobStoreFileName)) {
+        return "swift"
+    }
+    Properties properties = new Properties()
+    File propertiesFile = new File(blobStoreFileName)
+    propertiesFile.withInputStream {
+        properties.load(it)
+    }
+
+    return properties.getProperty("implementation", "swift")
+
+def startObjectStore() {
+    switch (readBlobStoreImplementation()) {
+        case "s3":
+            sh "docker run --detach=true --name=${containers.s3} ${images.s3}"
+            return rabbitmqS3Links
+        default:
```

Comment:
Almost done, one good practice for switch statements is to fail by default; then you will have to add a case for "swift", IMO.

But I agree if you don't take care of this comment.